.github/workflows/gradle-publish.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Publish package to the Maven Central Repository with Gradle
+
+env:
+  nexusUsername: ${{ secrets.SONATYPE_USER }}
```

Comment:
why not `NEXUS_USER`?

Reply:
Here we use Sonatype account for uploads to Nexus repository manager.

Reply:
Renamed to eliminate the confusion.